dlpar/dlpar_api/api.py

```diff
@@ -254,20 +418,37 @@ def get_cpu_option(self, linux_machine, option):
         o_cmd = 'lshwres -m ' + linux_machine.machine + \
                 ' --level lpar -r proc --filter lpar_names="' + \
                 linux_machine.partition + '" -F ' + option
-        opt_value = self.hmc.sshcnx.run_command(o_cmd).decode()
-
+        opt_value = self.hmc.sshcnx.cmd(o_cmd).stdout_text.strip()
         d_msg = option + ": " + opt_value + " for partition " + \
             linux_machine.partition
         self.log.debug(d_msg)
         return opt_value
 
+    def check_memory_proc(self, machine):
+        """
+        Check on a linux client how much physical memory we have.
+        """
+        memory = 0
+        p_cmd = "cat /proc/meminfo | grep MemTotal | awk -F' ' '{ print $2 }'"
```

Comment:
What is the need for this statement? I see the p_cmd value is overwritten in the try block below. Only one statement should suffice.

Reply:
This line is not needed, Removed in the latest commit.